### PR TITLE
Technical tweaks

### PR DESCRIPTION
**Pull request by slochower**
I made some grammar changes, as discussed.

I also updated the table format to allow larger figures, with equal size for common chemical features, fixed width font for the SMILES strings, automatic alignment of numbers with uncertainty, and other minor tweaks.

### CODE REVIEW

File: paper/benchmarkset.tex [hunk 1]
```diff
@@ -335,36 +339,49 @@ \subsubsection{Cucurbiturils}
...
 The water model also impacts the number of sodium ions which must be displaced (in sodium-based buffer) on binding~\cite{gao_binding_2015, henriksen_computational_2015}.
 
 Despite its apparent simplicity, CB7 is still a challenging benchmark that can put important issues into high relief. For example, in SAMPL4, free energy methods yielded $R^2$ values from 0.1 to 0.8 and RMS errors of about 1.9 to 4.9 kcal/mol for the same set of CB7 cases. This spread of results across rather similar methods highlights the need for shared benchmarks. Potential explanations include convergence difficulties, subtle methodological differences, and details of how the methods were applied~\cite{muddana_sampl4_2014}. Until the origin of such discrepancies is clear, it is difficult to know how accurate our methods truly are. 
 
+% Changes
+% Used booktabs for formatting (may not be completely compatible with revtex...).
+% Used siunitx to format the numbers and set automatic alignment with uncertainty.
```
Comment (davidlmobley) [inline paper/benchmarkset.tex:352]: This is beautiful; I'd never seen the SI unit module before. Definitely something to use again!

**Comment by davidlmobley** [comment]
Thanks, David, for the editorial fixes, and **huge thanks** for the table improvements!

**Comment by slochower** [comment]
Yes, if you haven't seen the documentation, it's quite powerful and comes in handy from time to time. One use case I have is to input raw numbers from python that may be in a format like 0.00000001 or 1e-8 and have the `siunitx` package automatically format everything into scientific notation, and when doing so, keep the same power of 10 for each number so I can make quick comparisons.
http://ctan.sharelatex.com/tex-archive/macros/latex/contrib/siunitx/siunitx.pdf

**Comment by slochower** [comment]
Not to derail this conversation too far, but this little snippet in the preamble is useful for our stuff, since molar isn't technically a SI unit.

```
\DeclareSIUnit\molar{\mole\per\cubic\deci\metre}
\DeclareSIUnit\Molar{\textsc{m}}
```

Then you can write `\si{1e-6}{\Molar}` in the text to get a little M as the unit or do unit transformations with `\molar`.

**Pull request merged**